docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,63 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.37  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents.  Where no service water-heating system exists or has been designed and submitted with design documents but the building will have service water-heating loads, a service water-heating system shall be modeled that matches the system type in the baseline building design"
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** custom evaluation context created for this rule of all SWH Distribution system ids located in the proposed, baseline and user RMDs  
+**Data Lookup:**   
+**Function Call:**  
+- **get_component_by_id**  
+- **compare_swh_dist_systems_and_components**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements  
+- **get_swh_components_associated_with_each_swh_distribution_system**  
+
+
+**Applicability Checks:**
+- check that the U_RMD has SWH Uses with loads
+
+## Applicability Checks:
+- in this case, all of the SWH Distribution IDs identified when the custom context was created are applicable to this rule.
+- Look at each swh distribution system id: `for swh_dist_system_id in context: CONTINUE TO RULE LOGIC`
+
+  ## Rule Logic #
+  - there are a couple cases here:
+    1.  If the swh distribution exists in the user model, AND has swh use, then expect user and proposed systems to be match
+    2.  Otherwise, expect the distribution system to match between proposed and baseline
+    3.  Systems that don't do 1 or 2 fail
```

Comment:
even though you are writing these as a numbered list in the "code", the markdown displays them as a) b) c)

I suggest changing to lettering in the "code" too so it is consistent